src/app/components/ConnectorDrawer/ConnectorDrawer.tsx

```diff
@@ -229,6 +231,25 @@ export const ConnectorDrawerPanelContent: FunctionComponent<ConnectorDrawerPanel
             <DrawerCloseButton onClick={onClose} />
           </DrawerActions>
         </DrawerHead>
+        {status === 'failed' && (
+          <Hint className="pf-u-mr-lg pf-u-ml-lg pf-u-p-md">
+            <HintBody>
+              <p>{t('previewModeMsg')}</p>
+              <Trans i18nKey={'supportEmailMsg'}>
+                You can still get help by emailing us at
+                <ClipboardCopy
+                  hoverTip="Copy"
+                  clickTip="Copied"
+                  variant="inline-compact"
+                >
+                  rhosak-eval-support@redhat.com
+                </ClipboardCopy>
+                . This mailing list is monitored by the Red Hat OpenShift
```

Comment:
@indraraj Can we use these texts with i18n?

Reply:
I am using here the `<Trans i18nKey={'supportEmailMsg'}>` to use the i18n, let me know if it's incorrect.

Reply:
Yeah, my bad.